src/frontend/utils/tests/render.tsx

```diff
@@ -0,0 +1,144 @@
+import {
+  act,
```

Comment:
In your commit message: `s/Create a custom render method to use in our tests wrapping the rended component/Create a custom render method to use in our tests wrapping the rendered component.`